tasks/cert.yaml

```diff
@@ -16,23 +14,23 @@
   register: _services_stopped
 
 - name: Obtain or renew cert for domain
-  shell: ./certbot-auto certonly --text -n --no-self-upgrade -m {{ letsencrypt_email }} --domains {{ _letsencrypt_domains | default(letsencrypt_domain) }} --agree-tos --standalone --expand {{_letsencrypt_certbot_args | join(' ')}} 2>&1
+  shell: ./certbot-auto {{_letsencrypt_certbot_combined_args | join(' ')}} 2>&1
   args:
     chdir: /opt/certbot
     executable: /bin/bash
   ignore_errors: true
   register: _certbot_command
 
 - set_fact: _signing_successful='{{ certbot_success_message in _certbot_command.stdout }}'
-- set_fact: _signing_skipped='{{ (letsencrypt_force_renew != true) and (certbot_skip_renewal_message in _certbot_command.stdout) }}'
+- set_fact: _signing_skipped='{{ ( not letsencrypt_force_renew) and (certbot_skip_renewal_message in _certbot_command.stdout) }}'
```

Comment:
Didn't change these -- just switched the from == false to treating it like a boolean